tests/unit/cli/test_utils.py

```
@@ -0,0 +1,81 @@
+
+    @pytest.mark.unit
+    def test_none_returns_current_month(self):
+        """Test that None returns current month."""
+        result = utils.parse_month_key(None)
+        assert len(result) == 7
+        assert result[4] == "-"
+
+    @pytest.mark.unit
+    def test_invalid_format_raises(self):
+        """Test that invalid format raises ValueError."""
+        with pytest.raises(ValueError):
```

Ruff (0.14.7)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)